hack/chains/setup-controller-certs.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Based on Christophe's procedure to get working SSL in a
+# local CRC cluster so the chains signed provenance demo works.
+# See https://docs.engineering.redhat.com/x/zFp-Dw .
+#
+# This is a temporary workaround. There should be a better way to
+# ensure the chains controller has access a set of CA certs -
+# maybe with https://cert-manager.io/ , or maybe the certs are
+# available already somewhere else in the cluster and just need
+# to be exposed to the chains controller.
+#
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source $SCRIPTDIR/_helpers.sh
+set -u
+
+TMP_DIR=$( mktemp -d )
+
+#
+# First prepare some CA certs and create a secret to hold them
+#
+
+# A cert from your local cluster to get local SSL working
+kubectl get secret router-ca -n openshift-ingress-operator -o jsonpath="{.data.tls\.crt}" | base64 -d > $TMP_DIR/cluster-ingress.pem
```

Comment:
Does not work with cluster bot, internal network is using different certificate. 
```
getting signed image: Get \"https://image-registry.openshift-image-registry.svc:5000/v2/\": x509: certificate signed by unknown authority
```

Reply:
Did the gitops change work for you in cluster bot? Perhaps I should split that into a separate PR so it could be merged sooner.

Reply:
Ack re the CA cert. Do you know where to find the right cert?